### PR TITLE
Completed T08

### PR DESCRIPTION
**Pull request by Ajay07Metamation**

### CODE REVIEW

File: Training/Program.cs [hunk 1]
```diff
@@ -1,7 +1,9 @@
-﻿namespace Training {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
+﻿// Tables
+Console.Write ("Till which tables do you want to print : ");
+int count = int.Parse (Console.ReadLine ());
+for (int i = 1; i <= count; i++) {
```
Comment (rajeevramanTMM) [inline Training/Program.cs:4]: Outer loop can be a while loop.

File: Training/Program.cs [hunk 2]
```diff
@@ -1,7 +1,9 @@
...
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
+﻿// Tables
+Console.Write ("Till which tables do you want to print : ");
+int count = int.Parse (Console.ReadLine ());
+for (int i = 1; i <= count; i++) {
+   for (int j = 1; j <= 10; j++) {
```
Comment (rajeevramanTMM) [inline Training/Program.cs:5]: If a loop has one statement no need of curly braces.

File: Training/Program.cs [hunk 3]
```diff
@@ -1,7 +1,9 @@
...
+      Console.WriteLine ($"{i}*{j} = {i * j}");
```
Comment (rajeevramanTMM) [inline Training/Program.cs:6]: Printing was not as expected. Please refer the image in the assignment doc file.

File: Training/Program.cs [hunk 4]
```diff
@@ -1,7 +1,10 @@
...
-    }
-}
\ No newline at end of file
+﻿// Tables
+using static System.Console;
+Console.Write ("Till which tables do you want to print : ");
+if (int.TryParse (Console.ReadLine () ?? "", out int count)) {
+   for (int i = 1; i <= count; i++) {
+      for (int j = 1; j <= 10; j++)
+         Console.WriteLine ($"{i,2} * {j,2} = {i * j,-1}");
```
Comment (rajeevramanTMM) [inline Training/Program.cs:7]: Why referring console again you have declared this has static in the using statement.

**Review by rajeevramanTMM — CHANGES_REQUESTED** [review]

File: Training/Program.cs (hunk 4 above)
Comment (Ajay07Metamation) [reply Training/Program.cs]: All changes done in latest commit

File: Training/Program.cs [hunk 5]
```diff
@@ -1,7 +1,10 @@
...
-            Console.WriteLine ("Hello, World!");
-        }
-    }
-}
\ No newline at end of file
+﻿// Tables
+using static System.Console;
+Write ("Till which tables do you want to print : ");
+if (int.TryParse (ReadLine () ?? "", out int count)) {
+   for (int i = 1; i <= count; i++) {
```
Comment (rajeevramanTMM) [inline Training/Program.cs:5]: I have mentioned to change the outer loop to while in previous review comment can you please check.

**Review by rajeevramanTMM — CHANGES_REQUESTED** [review]

File: Training/Program.cs (hunk 5 above)
Comment (Ajay07Metamation) [reply Training/Program.cs]: All changes done in latest commit

**Pull request closed without merging**